### DISCUSSION TITLE
How do I change the body style and class?

### DISCUSSION THREAD

**Discussion by ohmyohmyohmy** [jacobobryant/biff#229] [General]
Hi,

I can't seem to find the code that sets the style and class for the `<body>`? I thought it would be in ui/base, but I don't see where I can make the changes I want. How do I go about making changes to it please?

Thanks

**Comment by ohmyohmyohmy** [comment]
Nevermind. I think I've managed to find the solution here: https://github.com/jacobobryant/biff/discussions/194

Just gotta copy the source and make whatever changes I need

**Reply by jacobobryant** [reply]
Yes, that's what I'd recommend. I'll make a note to think about maybe redoing the way `biff/base-html` works so that it doesn't require copy-and-pasting to put attributes on `:body`.
